### PR TITLE
Tournament results page for round system

### PR DESCRIPTION
**Pull request by k-yat**

### CODE REVIEW

File: templates/admin/players.html.twig [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {% extends '@EasyAdmin/default/layout.html.twig' %}
 {% block content %}
+    <link rel="stylesheet" href="{{ asset('build/tournament.css') }}"/>
```
Comment (chabanenk0) [inline templates/admin/players.html.twig:3]: As far as I understand, it's enough to include js file, css file is already included in tournament.js. Please try removing this link. Stylesheets should be included in <head> not in <body> tag

File: templates/admin/round_system_results.html.twig [hunk 2]
```diff
@@ -0,0 +1,63 @@
+{% extends '@EasyAdmin/default/layout.html.twig' %}
+{% block content %}
+    <div class="container">
+        <h4>{{ tournament.title }}</h4>
+        <p>0 - чорні виграли, 1 - нічия, 2 - білі виграли</p>
```
Comment (chabanenk0) [inline templates/admin/round_system_results.html.twig]: please implement translations for this text

File: templates/admin/save_results.html.twig [hunk 3]
```diff
@@ -0,0 +1,87 @@
+{% extends '@EasyAdmin/default/layout.html.twig' %}
+{% block content %}
+    <link rel="stylesheet" href="{{ asset('build/tournament.css') }}">
```
Comment (chabanenk0) [inline templates/admin/save_results.html.twig:3]: The same: should work without this line, css should be included in tournament.js,

File: templates/admin/save_results.html.twig [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+
+
+    <div class="row list_of_participant">
+        <div class="col-md-2"></div>
+        <div class="col-md-10">
+            <table>
+                <tr>
+                    <td>Місце</td>
```
Comment (chabanenk0) [inline templates/admin/save_results.html.twig]: Translation is needed, probably it's not related to round robin system, but please implement trans if possible

File: templates/admin/save_results.html.twig [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+                        /
+                    </td>
+                </tr>
+            {% endfor %}
+            </table>
+        </div>
+    </div>
+
+    <script>
+        $('.res').on('click', function () {
```
Comment (chabanenk0) [inline templates/admin/save_results.html.twig:76]: Please use bootstrap dialogs. @MagnetosUA

File: src/Form/Type/RoundResultType.php [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+            ])
+            ->add('blackParticipantLastName', TextType::class, [
+                'label' => $this->translator->trans('Black participant last name'),
+                'attr' => [
+        'readonly' => true
+    ]
+            ])
+            ->add('result', ChoiceType::class, [
+        'choices' => [
+            'білі виграли' => RoundResult::RESULT_WHITE_WIN,
```
Comment (chabanenk0) [inline src/Form/Type/RoundResultType.php]: Please use translations for these phrases

File: templates/admin/round_system_results.html.twig [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+                    <td>{{ participant_one.participantOrder }}</td>
+                    <td>{{ participant_one.player.lastName }}</td>
+
+                    {% for participant_two in participants %}
+                        <td>
+                            {% if participant_one == participant_two %}
+                                -
+                            {% else %}
+
+                                <a class="roundResultShowButton">
```
Comment (chabanenk0) [inline templates/admin/round_system_results.html.twig]: It's preferable to name html classes like the following: round-result-show-button

File: templates/admin/round_system_results.html.twig [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+                                    ></i>
+                                </a>
+                            {% endif %}
+                        </td>
+                    {% endfor %}
+                </tr>
+            {% endfor %}
+            </tbody>
+        </table>
+        <div class="modal fade" id="setRoundResultModal" tabindex="-1" role="dialog"
```
Comment (chabanenk0) [inline templates/admin/round_system_results.html.twig]: the same for ids: round-result-modal

File: assets/js/app.js [hunk 9]
```diff
@@ -2,11 +2,11 @@
 
 var $ = require('jquery');
 
-require('select2');
-
 require('../css/app.css');
```
Comment (chabanenk0) [inline assets/js/app.js:7]: no need for these empty lines, please leave only one

File: assets/js/tournament.js [hunk 10]
```diff
@@ -0,0 +1,15 @@
...
+require('../css/tournament.css');
+var $ = require('jquery');
+
+require('select2');
+
+$(document).ready(function() {
+    $('.new-participant-dropdown').select2();
+});
+
+
```
Comment (chabanenk0) [inline assets/js/tournament.js:12]: remove extra new lines

File: templates/admin/round_system_results.html.twig (hunk 2 above)
Comment (k-yat) [reply templates/admin/round_system_results.html.twig]: done

File: templates/admin/save_results.html.twig (hunk 4 above)
Comment (k-yat) [reply templates/admin/save_results.html.twig]: done

File: src/Form/Type/RoundResultType.php (hunk 6 above)
Comment (k-yat) [reply src/Form/Type/RoundResultType.php]: done

File: templates/admin/round_system_results.html.twig (hunk 7 above)
Comment (k-yat) [reply templates/admin/round_system_results.html.twig]: done

File: templates/admin/round_system_results.html.twig (hunk 8 above)
Comment (k-yat) [reply templates/admin/round_system_results.html.twig]: done

**Review by chabanenk0 — APPROVED** [review]

**Pull request merged**